docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -1191,14 +1093,6 @@ The ListItem's `componentsProps` prop was deprecated in favor of `slotProps`:
  />
 ```
 
-### ContainerComponent
```

Comment:
@aarongarciah, here's the context of this PR: https://github.com/mui/material-ui/issues/41281#issuecomment-2140855314. `ContainerComponent` and `ContainerProps` is a special case as `secondaryAction` is deprecated, thus these prop will never be used. Should we keep these two deprecations? I think we should

Reply:
Yes, I think we should keep the deprecation for `ContainerComponent` and `ContainerProps`.

(note: what got deprecated was `ListItemSecondaryAction`, not `secondaryAction`, but the reasoning still stands)